nhentai/downloader.py

```diff
@@ -138,7 +139,8 @@ def start_download(self, queue, folder='', regenerate_cbz=False):
             logger.warning(f'Path "{folder}" already exist.')
 
         if os.getenv('DEBUG', None) == 'NODOWNLOAD':
-            return
+            # Assuming we want to continue with rest of process?
```

Comment:
In some cases we don't need to download files, use `export DEBUG=NODOWNLOAD` to skip rest process. In other cases we can use `export DEBUG=1` to enable debug mode and continue download process

Reply:
This should be fine then. Returning `True` will proceed with everything else in main, as it did previously.